packages/web/src/common/store/notifications/sagas.ts

```diff
@@ -145,6 +145,11 @@ export function* fetchNotifications(action: FetchNotifications) {
         getFeatureEnabled,
         FeatureFlags.SUPPORTER_DETHRONED_ENABLED
       )) as boolean | null) ?? false
+    const useDiscoveryNotifications =
```

Comment:
def a tragic situation with the double (( and casting... will make a note to improve those types

Reply:
yeah, unf and thx!